.github/travis/common.sh

```diff
@@ -51,6 +51,32 @@ if [ -z "$DATESTR" ]; then
 	fi
 fi
 
+function make_target() {
+  target=$1
+
+  if [ ! -v MAKE_JOBS ]; then
```

Comment:
Might make sense to just call this `CPUS`?

Reply:
If we want to override it I think this name makes more sense. Longer term, I think the way kokoro handles it may be more flexible. They could be unified, but don't think it's worth that effort at the moment.